.github/workflows/publish-site.yml

```diff
@@ -0,0 +1,41 @@
+name: Publish Site
+on:
+  push:
+      branches:
+        - master
+      paths:
+        - 'README.md'
+        - 'site/**'
+        - '!site/update.sh'
+        - '!site/site.md'
+
+jobs:
+  main:
+    name: Publish Site
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+
+    steps:
+      - uses: linz/action-typescript@v3
+
+      - name: AWS Configure
+        uses: aws-actions/configure-aws-credentials@v2
+        with:
+          aws-region: ap-southeast-2
+          mask-aws-account-id: true
+          role-to-assume: ${{ secrets.AWS_CI_ROLE }}
+
+      - name: AWS Configure ODR
+        uses: aws-actions/configure-aws-credentials@v2
+        with:
+          aws-region: ap-southeast-2
+          mask-aws-account-id: true
+          role-to-assume: ${{ secrets.AWS_ODR_DATAMANAGER_ROLE }}
+          role-chaining: true
+
+      - name: Deploy page to S3
+        run: |
+          node site/generate.site.mjs
+          aws s3 cp index.html s3://nz-imagery/index.html
```

Comment:
I think this would need a `--content-type` or something?

Reply:
The content-type is already guessed with the right value:
```
aws s3api head-object --bucket linz-imagery-test --key index.html
{
    "AcceptRanges": "bytes",
    "LastModified": "2023-09-20T01:34:51+00:00",
    "ContentLength": 3542,
    "ETag": "\"bdeac7eb9b32a1d3140329b7fc15b3b6\"",
    "ContentType": "text/html",
    "ServerSideEncryption": "AES256",
    "Metadata": {}
}
```

I can still force it.